wagtail/admin/panels/page_utils.py

```diff
@@ -31,7 +32,7 @@ def set_default_page_edit_handlers(cls):
     cls.promote_panels = [
         MultiFieldPanel(
             [
-                FieldPanel("slug"),
+                FieldPanel("slug", widget=SlugInput()),
```

Comment:
```suggestion
                FieldPanel("slug", widget=SlugInput),
```

No need to call (instantiate) the class here, Django will do this for us.